libr/core/carg.c

```diff
@@ -193,16 +191,7 @@ R_API void r_core_print_func_args(RCore *core) {
 				argcnt++;
 			}
 		} else {
```

Comment:
What about fixing the function or removing it completely?

Reply:
I think the void fallback like this is ok for the current implementation.

Reply:
what about printing nargs = 4 so at least we can get the values? anyway im fine for removing this beacuse all this stuff needs to be refactored and cleaned up

Reply:
sgtm

Reply:
So wat do

Reply:
Nothing, just did it